### PR TITLE
Translation for Crosswalk Download Mode & Lite

### PR DESCRIPTION
**Pull request by jondong**
@bspencer PTAL, thanks

### CODE REVIEW

**Comment by baleboy** [comment]
can you also add the _zh entries to public/documentation/_data.json ? Right now the pages you are adding are not listed in the sidebar.

**Comment by jondong** [comment]
@baleboy Thanks for the hint, updated, PTAL again.

File: public/documentation/_data.json [hunk 1]
```diff
@@ -78,6 +78,10 @@
   "download_mode": {
     "title": "Download Mode"
   },
+  "download_mode_zh": {
+    "title": "下载模式",
+    "origin": "download_mode"
+  },
   "crosswalk_lite": {
```
Comment (baleboy) [inline public/documentation/_data.json]: you still need crosswalk_lite_zh here

**Comment by jondong** [comment]
@baleboy thanks for the remind, updated, please take a look again.

**Pull request merged**